common/v2/features/DeFiZap/components/ZapForm.tsx

```diff
@@ -40,85 +103,116 @@ const ZapForm = (props: Props) => {
+          });
+        }}
+        render={({ values, errors, touched, setFieldValue }) => {
+          const handleNonceEstimate = async (account: IAccount) => {
+            const nonce: number = await getNonce(values.network, account);
+            setFieldValue('nonce', nonce);
+          };
+          const isValid =
+            Object.values(errors).filter(error => error !== undefined && !isEmpty(error)).length ===
+            0;
+
+          return (
+            <Form>
+              {zapSelected && <ZapSelectedBanner zapSelected={zapSelected} />}
+              <FormFieldItem>
+                <FormFieldLabel htmlFor="account">{translate('X_SENDER')}</FormFieldLabel>
+                <Field
+                  name="account"
+                  value={values.account}
+                  component={({ field, form }: FieldProps) => (
+                    <AccountDropdown
+                      name={field.name}
+                      value={field.value}
+                      accounts={relevantAccounts}
+                      onSelect={(option: IAccount) => {
+                        form.setFieldValue('account', option); //if this gets deleted, it no longer shows as selected on interface, would like to set only object keys that are needed instead of full object
+                        handleNonceEstimate(option);
+                      }}
+                    />
+                  )}
+                />
+              </FormFieldItem>
+              <FormFieldItem>
+                <FormFieldLabel htmlFor="amount">
+                  <div>{translate('SEND_ASSETS_AMOUNT_LABEL')}</div>
+                </FormFieldLabel>
+                <Field
+                  name="amount"
+                  validate={validateAmountField}
+                  render={({ field, form }: FieldProps) => {
+                    return (
+                      <>
+                        <AmountInput
+                          {...field}
+                          asset={ethAsset}
+                          value={field.value}
+                          onBlur={() => {
+                            form.setFieldTouched('amount');
+                            //handleGasEstimate();
+                          }}
+                          placeholder={'0.00'}
+                        />
+                        {errors && errors.amount && touched && touched.amount ? (
+                          <InlineMessage className="SendAssetsForm-errors">
+                            {errors.amount}
+                          </InlineMessage>
+                        ) : null}
+                      </>
+                    );
+                  }}
+                />
+              </FormFieldItem>
+              <FormFieldSubmitButton
+                type="submit"
+                onClick={() => {
+                  if (isValid) {
+                    onComplete(values);
+                  }
+                }}
+              >
+                Continue on!
```

Comment:
Use translation

Reply:
We're going to go back over the whole flow and do translations at the end of it when we can have design review copy all in one go c: